backend/assignments/1/src/main/java/shrishti/example/Main.java

```diff
@@ -0,0 +1,92 @@
+package shrishti.example;
+import java.io.BufferedReader;
+
+import java.nio.file.Path;
+
+import java.util.*;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Main {
+    public static void main(String[] args) {
+
+    }
+    public static ArrayList<String[]> parseCSV(Path path) {
```

Comment:
More modularity expected.
Move utils, core logic etc in seperate classes/packages